app/jenkins-for-jira-ui/src/components/ConnectJenkins/ConfigurationSteps/ConfigurationSteps.tsx

```diff
@@ -9,6 +10,17 @@ type AppProps = {
 	currentStage: string;
 };
 
+const mapCreateStatus = (currentStage: unknown): Status => {
```

Comment:
Why is this of type `unknown`? Don't we know that this is a string?

Reply:
Not sure why this was `unknown`. @rachellerathbone wrote this originally so I assumed it was an AtlasKit quirk. Looking at it now and `string` works correctly. Updated the code.